packages/opc-feedback/demo/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>opc-feedback</title>
+    <link type="text/css" rel="stylesheet" href="https://unpkg.com/@patternfly/patternfly/patternfly.css"
+        crossorigin="anonymous" />
+    <link type="text/css" rel="stylesheet" href="https://unpkg.com/@patternfly/patternfly/patternfly-addons.css"
+        crossorigin="anonymous" />
+    <script src="https://unpkg.com/@webcomponents/webcomponentsjs@2.0.0/webcomponents-loader.js"></script>
+    <script type="module" src="https://cdnjs.cloudflare.com/ajax/libs/ionicons/5.2.3/ionicons/ionicons.esm.js"></script>
+    <script nomodule="" src="https://cdnjs.cloudflare.com/ajax/libs/ionicons/5.2.3/ionicons/ionicons.js"></script>
+    <script src="opc-feedback.js"></script>
+
+    <style>
+        .opc-feedback-blue opc-feedback {
```

Comment:
i dont think we need to provide color theming like this. if you want to provide themese with color you need to use a theme property to ur component and provide the color there. check opc-header for themes

Reply:
Implemented the themes for ```red``` and ```blue``` color